csharp/tools/tracing-config.lua

```diff
@@ -48,6 +49,14 @@ function RegisterExtractorPack(id)
                     dotnetRunNeedsSeparator = true
                     dotnetRunInjectionIndex = i + 1
                 end
+                if arg == 'test' then
+                    testMatch = true
```

Comment:
So `dotnet run` when provided a dll or exe will not do any compilation?

Reply:
Sorry, I was maybe not clear: I intended to keep the `testMatch` variable, so specifically on this line it would say
```lua
testMatch = true
match = true
```
and then keep the check `if testMatch and (arg:match('%.exe$') or arg:match('%.dll'))` instead of just `if arg:match('%.exe$') or arg:match('%.dll')`

Reply:
> So `dotnet run` when provided a dll or exe will not do any compilation?

By closer inspection, that part is actually unclear as it is not documented as a part of `dotnet run`.
We will make sure to narrow the fix such that it only applies to `dotnet test` (which was a misunderstanding between @hvitved and me)